src/app/settings/components/settings/settings-menu/settings-menu.component.scss

```diff
@@ -18,10 +17,36 @@
       font-size: medium;
       display: block;
       text-decoration: none;
-      color: var(--primary-color);
+      color: var(--black-color);
       &.active-tab {
+        color: var(--white-color);
         background-color: var(--background-color);
-        color: var(--primary-color);
+      }
+    }
+  }
+
+  &__nav-submenu {
+    width: 100%;
+    font-weight: lighter;
+    padding-top: 0;
+    a {
+      position: relative;
+      display: block;
+      padding: 12px 16px 12px 30px;
+      font-size: medium;
+      text-decoration: none;
+      color: var(--black-color);
+      &.active-tab {
+        font-weight: 400;
+        text-decoration: underline;
+        &::before {
+          content: ' \203A';
+          position: absolute;
```

Comment:
What are using position absolute and relative for here? Seems unnecessary

Reply:
Not really it's remove it.